contracts/Zauction.sol

```diff
@@ -35,30 +37,44 @@ contract ZAuction is Initializable, OwnableUpgradeable {
     uint256 amount,
     address nftAddress,
     uint256 tokenId,
-    uint256 expireBlock
+    uint256 expireBlock,
+    address paymentToken,
+    uint256 topLevelDomainId
   );
 
   event DomainSold(
     address indexed buyer,
     address indexed seller,
     uint256 amount,
     address nftAddress,
-    uint256 indexed tokenId
+    uint256 indexed tokenId,
+    address paymentToken,
+    uint256 topLevelDomainId
```

Comment:
Make a new `DomainSoldV2` event instead of updating the old one. Same applies with `BuyNowPriceSet`

Reply:
I don't think we need `BuyNowPriceSet` more than once. The other two make sense because we need to support accepting legacy bids, or allowing accepting of legacy listings in `buyNow`. Calls to set a buy price going forward will only support setting that buyPrice with v2.1 changes, so we only need one

Reply:
We need two. Subgraph needs to be able to read events that have already been emitted.